scipy/interpolate/fitpack.py

```diff
@@ -310,7 +310,7 @@ def splev(x, tck, der=0, ext=0):
         of the spline. (Also see Notes.)
     der : int, optional
         The order of derivative of the spline to compute (must be less than
-        or equal to k).
+        or equal to the degree of the spline).
```

Comment:
Maybe better say `equal to k, the degree of the spline`.

Reply:
Sure! Done.